producer_client/spec/pacts/animals_client-animals_service.json

```diff
@@ -20,13 +20,16 @@
         "body": {
           "cats": [
             {
-              "name": "Garfield"
+              "name": "Garfield",
+              "age": 39
             },
             {
-              "name": "Felix"
+              "name": "Felix",
+              "age": 99
             },
             {
-              "name": "Sylvester"
+              "name": "Sylvester",
+              "age": 76
```

Comment:
The client is made age-aware so that it's contract with the producer is thorough.